replacements.js

```diff
@@ -324,5 +328,1207 @@ module.exports = [
+	['ޭ', 'ey'],
+	['ޮ', 'o'],
+	['ޯ', 'oa'],
+	['ް', ''],
+
+	// Greek
+	['α', 'a'],
+	['β', 'v'],
+	['γ', 'g'],
+	['δ', 'd'],
+	['ε', 'e'],
+	['ζ', 'z'],
+	['η', 'i'],
+	['θ', 'th'],
+	['ι', 'i'],
+	['κ', 'k'],
+	['λ', 'l'],
+	['μ', 'm'],
+	['ν', 'n'],
+	['ξ', 'ks'],
+	['ο', 'o'],
+	['π', 'p'],
+	['ρ', 'r'],
+	['σ', 's'],
+	['τ', 't'],
+	['υ', 'y'],
+	['φ', 'f'],
+	['χ', 'x'],
+	['ψ', 'ps'],
+	['ω', 'o'],
+	['ά', 'a'],
+	['έ', 'e'],
+	['ί', 'i'],
+	['ό', 'o'],
+	['ύ', 'y'],
+	['ή', 'i'],
+	['ώ', 'o'],
+	['ς', 's'],
+	['ϊ', 'i'],
+	['ΰ', 'y'],
+	['ϋ', 'y'],
+	['ΐ', 'i'],
+	['Α', 'A'],
+	['Β', 'B'],
+	['Γ', 'G'],
+	['Δ', 'D'],
+	['Ε', 'E'],
+	['Ζ', 'Z'],
+	['Η', 'I'],
+	['Θ', 'TH'],
+	['Ι', 'I'],
+	['Κ', 'K'],
+	['Λ', 'L'],
+	['Μ', 'M'],
+	['Ν', 'N'],
+	['Ξ', 'KS'],
+	['Ο', 'O'],
+	['Π', 'P'],
+	['Ρ', 'R'],
+	['Σ', 'S'],
+	['Τ', 'T'],
+	['Υ', 'Y'],
+	['Φ', 'F'],
+	['Χ', 'X'],
+	['Ψ', 'PS'],
+	['Ω', 'O'],
+	['Ά', 'A'],
+	['Έ', 'E'],
+	['Ί', 'I'],
+	['Ό', 'O'],
+	['Ύ', 'Y'],
+	['Ή', 'I'],
+	['Ώ', 'O'],
+	['Ϊ', 'I'],
+	['Ϋ', 'Y'],
+
+	// Latvian
+	['ā', 'a'],
+	['ē', 'e'],
+	['ģ', 'g'],
+	['ī', 'i'],
+	['ķ', 'k'],
+	['ļ', 'l'],
+	['ņ', 'n'],
+	['ū', 'u'],
+	['Ā', 'A'],
+	['Ē', 'E'],
+	['Ģ', 'G'],
+	['Ī', 'I'],
+	['Ķ', 'k'],
+	['Ļ', 'L'],
+	['Ņ', 'N'],
+	['Ū', 'U'],
+
+	// Macedonian
+	['Ќ', 'Kj'],
+	['ќ', 'kj'],
+	['Љ', 'Lj'],
+	['љ', 'lj'],
+	['Њ', 'Nj'],
+	['њ', 'nj'],
+	['Тс', 'Ts'],
+	['тс', 'ts'],
+
+	// Polish
+	['ą', 'a'],
+	['ć', 'c'],
+	['ę', 'e'],
+	['ł', 'l'],
+	['ń', 'n'],
+	['ś', 's'],
+	['ź', 'z'],
+	['ż', 'z'],
+	['Ą', 'A'],
+	['Ć', 'C'],
+	['Ę', 'E'],
+	['Ł', 'L'],
+	['Ń', 'N'],
+	['Ś', 'S'],
+	['Ź', 'Z'],
+	['Ż', 'Z'],
+
+	// Ukranian
+	['Є', 'Ye'],
+	['І', 'I'],
+	['Ї', 'Yi'],
+	['Ґ', 'G'],
+	['є', 'ye'],
+	['і', 'i'],
+	['ї', 'yi'],
+	['ґ', 'g'],
+
+	// Serbian
+	['ђ', 'dj'],
+	['ј', 'j'],
+	['ћ', 'c'],
+	['џ', 'dz'],
+	['Ђ', 'Dj'],
+	['Ј', 'j'],
+	['Ћ', 'C'],
+	['Џ', 'Dz'],
+
+	// Slovak
+	['ľ', 'l'],
+	['ĺ', 'l'],
+	['ŕ', 'r'],
+	['Ľ', 'L'],
+	['Ĺ', 'L'],
+	['Ŕ', 'R'],
+
+	// Hungarian
+	['ä', 'a'],
+	['Ä', 'A'], 
+	['ö', 'o'],
+	['Ö', 'O'], 
+	['ü', 'u'],
+	['Ü', 'U'],
+	['ű', 'u'],
+	['Ű', 'U'],
+
+	// Lithuania
+	['ą', 'a'],
+	['č', 'c'],
+	['ę', 'e'],
+	['ė', 'e'],
+	['į', 'i'],
+	['š', 's'],
+	['ų', 'u'],
+	['ū', 'u'],
+	['ž', 'z'],
+	['Ą', 'A'],
+	['Č', 'C'],
+	['Ę', 'E'],
+	['Ė', 'E'],
+	['Į', 'I'],
+	['Š', 'S'],
+	['Ų', 'U'],
+	['Ū', 'U'],
+
+	// Litvian
+	['ā', 'a'],
+	['č', 'c'],
+	['ē', 'e'],
+	['ģ', 'g'],
+	['ī', 'i'],
+	['ķ', 'k'],
+	['ļ', 'l'],
+	['ņ', 'n'],
+	['š', 's'],
+	['ū', 'u'],
+	['ž', 'z'],
+	['Ā', 'A'],
+	['Č', 'C'],
+	['Ē', 'E'],
+	['Ģ', 'G'],
+	['Ī', 'i'],
+	['Ķ', 'k'],
+	['Ļ', 'L'],
+	['Ņ', 'N'],
+	['Š', 'S'],
+	['Ū', 'u'],
+	['Ž', 'Z'],
+
+	// Swedish
+	['ä', 'a'],
+	['Ä', 'A'],
+	['ö', 'o'],
+	['Ö', 'O'],
+
+	// Slovak
+	['ä', 'a'],
+	['Ä', 'A'],
+
+	// Serbian
+	['љ', 'lj'],
+	['њ', 'nj'],
+	['Љ', 'Lj'],
+	['Њ', 'Nj'],
+	['đ', 'dj'],
+	['Đ', 'Dj'],
 ];
+
+
+'use strict';
+
+module.exports = [
```

Comment:
This is the second module.exports. Seems like you have copy-pasted things twice.